### PR TITLE
Add introduction to aliases

### LINKED ISSUE #22: Please add a section of aliases to the SQL lessons
This is just a reminder to do so.
Cheers!

### PR DESCRIPTION
**Pull request by ethanwhite**
This serves as a good foundation for more advanced SQL material and
aliases are very common in SQL so having been exposed to them will
help students understand material they will encounter online.

This material brought to you by the suggestion of @lwasser and @emhart.

Fixes #22.

### CODE REVIEW

**Comment by emhart** [comment]
Looks good Ethan.  Is it confusing to note that aliases don't require as?  Often I see queries (and write them) like 

```SQL
SELECT * FROM 
my.db a
JOIN
your.db b
ON a.id = b.id
```

I think there's a slight tension between 'here's how best to do things' and 'Here's how things are actually done in practice even if that isn't the best practice'.

File: sql.md [hunk 1]
```diff
@@ -375,6 +375,25 @@ could do something like
...
 
+Aliases
+-------
+
+As queries get more complex names can get long and unwieldy. To help make things
+clearer we can use aliases to assign new names to things in the query.
+
+We can alias both table names:
+
+  SELECT surv.year, surv.month, surv.day, surv.genus, sp.species
```
Comment (fmichonneau) [inline sql.md]: sp.species -> sp.species_id ?

File: sql.md [hunk 2]
```diff
@@ -375,6 +375,25 @@ could do something like
...
+  FROM surveys AS surv
+  JOIN species AS sp ON surv.species_id = sp.species_id
+
+And column names:
+
+  SELECT surv.year AS yr, surv.month AS mo, surv.day AS day, surv.genus AS gen, sp.species AS sp
```
Comment (fmichonneau) [inline sql.md]: same here

**Comment by ethanwhite** [comment]
I added a note on dropping `AS` and fixed the column names.

**Comment by ethanwhite** [comment]
Actually, those uses of `species` were correct because we want to display the species name not the species ID. I've reverted that change and kept the addition of the `AS` section.

**Comment by lwasser** [comment]
This is great. +1:

**Pull request merged**